IFC4x3/Properties/a/AreaID_1LX3rTqy19RRHx3VNZyWTI/DocProperty.xml

```diff
@@ -0,0 +1,2 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="AreaID_1LX3rTqy19RRHx3VNZyWTI" Name="AreaID" UniqueId="55843d5d-d3c0-496d-b47b-0df5e3f20752" PrimaryDataType="IfcLabel" />
```

Comment:
How about `IfcIdentifier`? Does it make sense or is it purely human readable?

Reply:
See previous comment.